assets/overlays/aws-ebs/aws_ebs.go

```diff
@@ -0,0 +1,72 @@
+package aws_ebs
+
+import (
+	"github.com/openshift/csi-operator/pkg/config/common"
+	"github.com/openshift/csi-operator/pkg/generator"
+)
+
+// GetAWSEBSGeneratorConfig returns configuration for generating assets of  AWS EBS CSI driver operator.
+func GetAWSEBSGeneratorConfig() *generator.CSIDriverGeneratorConfig {
+	return &generator.CSIDriverGeneratorConfig{
+		AssetPrefix:      "aws-ebs-csi-driver",
+		AssetShortPrefix: "ebs",
+		DriverName:       "ebs.csi.aws.com",
+
+		ControllerConfig: &generator.ControlPlaneConfig{
+			DeploymentTemplateAssetName: "overlays/aws-ebs/patches/controller_add_driver.yaml",
+			LivenessProbePort:           10301,
+			MetricsPorts: []generator.MetricsPort{
+				{
```

Comment:
Why is this an array? Configuration here I feel defines configuration of the driver. Can we have more than one of these in some cases?

Reply:
At least vSphere CSI driver has two containers and each of them can have its own metric port. We don't scrape syncer metrics, but perhaps we should.